src/main/java/uk/gov/companieshouse/itemhandler/model/ItemType.java

```diff
@@ -1,16 +1,96 @@
 package uk.gov.companieshouse.itemhandler.model;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.itemhandler.service.ChdItemSenderService;
+import uk.gov.companieshouse.itemhandler.service.EmailService;
+import uk.gov.companieshouse.kafka.exceptions.SerializationException;
+
+import javax.annotation.PostConstruct;
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutionException;
+
 public enum ItemType {
     CERTIFICATE("item#certificate"),
-    CERTIFIED_COPY("item#certified-copy");
+    CERTIFIED_COPY("item#certified-copy"),
+    MISSING_IMAGE_DELIVERY("item#missing-image-delivery") {
+        @Override
+        public void sendMessages(OrderData order)
+                throws InterruptedException, ExecutionException, SerializationException, JsonProcessingException {
+            getItemSender().sendItemsToChd(order);
+        }
+    };
+
+    private static final Map<String, ItemType> TYPES_BY_KIND;
+
+    static {
+        final Map<String, ItemType> map = new ConcurrentHashMap<>();
+        for (final ItemType type: ItemType.values()) {
+            map.put(type.getKind(), type);
+        }
+        TYPES_BY_KIND = Collections.unmodifiableMap(map);
+    }
 
     ItemType(final String kind) {
         this.kind = kind;
     }
 
+    public static ItemType getItemType(final String kind) {
+        return TYPES_BY_KIND.get(kind);
+    }
+
+    @Component
+    public static class Injector {
+
+        @Autowired
+        private EmailService emailer;
+
+        @Autowired
+        private ChdItemSenderService itemSender;
+
+        @PostConstruct
+        public void postConstruct() {
+            for (final ItemType type : EnumSet.allOf(ItemType.class)) {
+                type.setEmailer(emailer);
+                type.setItemSender(itemSender);
+            }
+        }
+    }
+
     private String kind;
+    private EmailService emailer;
+    private ChdItemSenderService itemSender;
 
     public String getKind(){
         return this.kind;
     }
+
+    /**
+     * Sends outbound Kafka message(s) to propagate the order and/or its items onwards for further processing.
+     * @param order the order to be propagated
+     * @throws InterruptedException
+     * @throws ExecutionException
+     * @throws SerializationException
+     * @throws JsonProcessingException
+     */
+    public void sendMessages(final OrderData order)
```

Comment:
This method name could perhaps be singular while its dealing with a single `OrderData`?

Reply:
You're right, for now, only a single outgoing message will result from an order, regardless of whether that message is bound for email or CHPRD. I had given this method a name suggesting there could be more than one outgoing message, because the idea is that post-MVP, we could have a number of items in the order, requiring a separate outgoing message to CHPRD for each item in the order.